src/subgraph/common/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Chain } from '@parifi/references';
 
 export const publicSubgraphEndpoints: { [key in Chain]: string } = {
-  [Chain.ARBITRUM_SEPOLIA]: 'https://api.thegraph.com/subgraphs/name/parifi/parifi-sepolia',
+  [Chain.ARBITRUM_SEPOLIA]: 'https://subgraph.satsuma-prod.com/ac10c1d41dcb/parifis-team--3804602/parifi-snx-sepolia/api',
```

Comment:
Use env variables for private endpoints. If public endpoints are not available, return empty string. 
Avoid using private endpoints in code directly